handbook/product/roadmap.md

```diff
@@ -74,7 +74,28 @@ See [Distribution roadmap](https://github.com/sourcegraph/about/pull/1104).
 
 ## Search
 
-TO DO
+1. 🔄 Enterprise homepage
```

Comment:
How does this relate to and/or intersect with goals on the search team page?

Reply:
The in-progress items map directly to the [goals](https://about.sourcegraph.com/handbook/engineering/search#goals) listed on the search team page.